app/design/frontend/Chervinskiy/drexel/Magento_Newsletter/web/css/source/_extend.less

```diff
@@ -0,0 +1,359 @@
+& when (@media-common = true) {
+
+    // Social links footer
+    .list-inline {
+        display: flex;
+        gap: 40px;
+
+        a,
+        li {
+            .lib-font-size(12);
+            text-transform: uppercase;
+            letter-spacing: 1px;
+        }
+    }
+
+    .footer-social {
+        display: flex;
+        gap: 50px;
+        justify-content: flex-end;
+
+        p {
+            text-transform: uppercase;
+            color: @color-drexel_subtitle;
+        }
+
+        .fa-brands {
+            .lib-font-size(17);
+        }
+    }
+
+    // Newsletter styling and "Subscribe" button
+    .newsletter {
+        display: flex;
+        justify-content: space-between;
+
+        .brand-title {
+            margin-left: 80px;
+
+            h2 {
+                margin-bottom: 0;
+                font-family: 'Poppins',sans-serif;
```

Comment:
Use a variable (@font-family__base)

Reply:
Done, also I created new variables for custom fonts and replace it in the whole project